autoware_lanelet2_extension/lib/bus_stop_area.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOLINTBEGIN(readability-identifier-naming)
+
+#include "autoware_lanelet2_extension/regulatory_elements/bus_stop_area.hpp"
+
+#include <boost/variant.hpp>
+
+#include <lanelet2_core/primitives/RegulatoryElement.h>
+
+#include <algorithm>
+#include <memory>
+#include <utility>
+#include <vector>
+
+namespace lanelet::autoware
+{
+namespace
+{
+template <typename T>
+bool findAndErase(const T & primitive, RuleParameters * member)
+{
+  if (member == nullptr) {
+    std::cerr << __FUNCTION__ << ": member is null pointer";
```

Comment:
I think we shouldn't use the C++ standard output methods. `console_bridge` is the alternative if you don't actually have a ROS environment.
https://github.com/ros2/console_bridge_vendor

Reply:
Exactly, I removed that cerr line because the `member` variable passed by the user being `nullptr` is not business logic error and makes sense to just return false. In future PR I'll fix the same lines in other source files

fixed in 61ed64f2e6a09cb15e5d4f542207e03514f792b1